website/docs/components/carousel/carousel-code.md

```diff
@@ -45,8 +45,8 @@ tabs: Design('carousel'), A11y('carousel-a11y'), API('carousel-api'), Example('c
 
 ## Hidden controls to move focus in hidden block
```

Comment:
```suggestion
## Hiding nested elements on hidden slides
```
something like that, if I understood correctly, because the title isn't clear right now